app/templates/index.hbs

```diff
@@ -2,6 +2,19 @@
   <h1>Welcome to my site!</h1>
 </div>
 
+{{#if this.isDev}}
+<div class='qrCodeContainer'> 
+<div class='qrSection'>
+  <QrCode
+    @text='http://example.com'
+    class='qrcode'
+    @darkColor='#00000'
```

Comment:
Can we make a separate component for this QR code feature?

Reply:
sure